public/app/features/alerting/unified/components/admin/ExternalAlertmanagers.tsx

```diff
@@ -188,20 +237,24 @@ export const ExternalAlertmanagers = () => {
               })}
             </tbody>
           </table>
-          <div>
-            <Field
-              label="Send alerts to"
-              description="Sets which Alertmanager will handle your alerts. Internal (Grafana built in Alertmanager), External (All Alertmanagers configured above), or both."
-            >
-              <RadioButtonGroup
-                options={alertmanagerChoices}
-                value={alertmanagersChoice}
-                onChange={(value) => onChangeAlertmanagerChoice(value!)}
-              />
-            </Field>
-          </div>
         </>
       )}
+
+      {hasExternalAlertmanagers && (
+        <div className={styles.amChoice}>
```

Comment:
The way that the elements are arranged it makes it unclear whether this option is going to be removed or not - I'd consider moving this section right after the "Alertmanager data sources" section to make it a bit more clear.